spec/math_spec.rb

```diff
@@ -1060,4 +1060,13 @@
       end
     end
   end
+
+  context "#positive_definite? for #{dtype}" do
```

Comment:
This is not working because you've added it outside the dtype context block. Please have a closer look at the file structure. You can also test your specs on your own machine by doing `bundle exec rake spec`.